track_and_trace/asset_client/utils/submit_utils.js

```diff
@@ -0,0 +1,342 @@
+/**
```

Comment:
This looks like a lot of duplicate code? Why doesn't it use the existing transaction submission module?

Reply:
the existing one utilizes mithril requests which aren't available when this script is run.

Reply:
Ok. Lets put this in the scripts directory to make it clear that it is only for the scripts.

Reply:
updated